src/sentry/workflow_engine/tasks.py

```diff
@@ -111,6 +113,46 @@ def fetch_event(event_id: str, project_id: int) -> Event | None:
     )
 
 
+def build_workflow_event_data(
```

Comment:
nit: this is specifically building a `GroupEvent` based workflow event data -- with the activity changes we might want to give this a more specific name around that.